server/pulp/server/webservices/middleware/postponed.py

```diff
@@ -31,39 +18,12 @@ def _get_operation_postponed_body(exception):
         report = TaskResult.from_async_result(exception.call_report)
```

Comment:
This entire method is only used by the `DjangoPostponedOperationMiddleware` class. Please move it into that class as a `@staticmethod`. That will keep it out of the modules namespace.